frontend/src/metabase/actions/containers/ActionExecuteModal/ActionExecuteModal.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useCallback } from "react";
+import { t } from "ttag";
+import { connect } from "react-redux";
+import _ from "underscore";
+import Actions from "metabase/entities/actions";
+import ModalContent from "metabase/components/ModalContent";
+import {
+  ActionFormSubmitResult,
+  ParametersForActionExecution,
+  WritebackAction,
+  WritebackActionId,
+} from "metabase-types/api";
+import { State } from "metabase-types/store";
+import { executeAction, ExecuteActionOpts } from "../../actions";
+import ActionParametersInputForm from "../ActionParametersInputForm";
+
+interface OwnProps {
+  actionId: WritebackActionId;
+  onSubmit: (opts: ExecuteActionOpts) => Promise<ActionFormSubmitResult>;
+  onClose?: () => void;
+}
+
+interface ActionLoaderProps {
+  action: WritebackAction;
+}
+
+type ActionExecuteModalProps = OwnProps & ActionLoaderProps;
+
+const ActionExecuteModal = ({
+  action,
+  onSubmit,
+  onClose,
+}: ActionExecuteModalProps) => {
+  const title =
+    action.parameters.length > 0 ? action.name : t`Run ${action.name}?`;
```

Comment:
I wonder if we can tweak / use [`getFormTitle` from `metabase/actions/utils`](https://github.com/metabase/metabase/blob/c7a19a4040c2ee3fff1e669d77a61764f7a6d1c5/frontend/src/metabase/actions/utils.ts#L223) here somehow?